### ISSUE TITLE
New Ontology anchor metaschema

### ISSUE THREAD

**Issue by netfl0** [ocsf/ocsf-schema#1218]
TLDR:
> Create a new metaschema object to support anchoring OCSF concepts in ontology. The primary goal is to enable richer and deeper programmatic integration, and ontological reasoning between the OCSF and D3FEND ontology projects.

Ontology and schema development are complimentary and essential. The schema project moves fast, it captures the information needs of leading edge users, technology developers, and practitioners into a **conceptual** model. The ontology development compliments this by integrating these needs and concepts into a comprehensive **ontological** model of [Digital Systems](https://d3fend.mitre.org/dao/artifact/d3f:DigitalSystem/). This has many use cases, but foremost model-based systems engineering and ontological model validation. 

We believe we need the expressiveness, at a minimum, of [OWL2-DL Axioms](https://www.w3.org/TR/owl2-syntax/#Axioms) to drive ontological systems model validation.

Additional goals:
* D3FEND Ontology adopts OCSF properties on D3FEND Classes, especially forthcoming [Event Classes](https://github.com/d3fend/d3fend-ontology/issues/309)
* Drive insights from ontology modeling back into OCSF Event to Object relationships
* Align schema and knowledge base efforts to increase practitioner understanding and technical knowledge.

The integration of OCSF and D3FEND ontology, in the manner prescribed by this eventual pull request, will enable us to take real world collections of events and **_infer_** a complex systems model for analysis. There is considerable work to get to this vision, but we believe the combination of the open source [D3FEND ontology](https://github.com/d3fend/d3fend-ontology) and the open source OCSF are the most promising path forward.

**Comment by pagbabian-splunk** [comment]
Addressed via the `references` keyword.
